README.md

```diff
@@ -0,0 +1 @@
+[![Build Status](https://dev.azure.com/event-maker-telegram-bot/Core/_apis/build/status/Validate%20the%20code?branchName=feature%2Fissue-2-add-azure-pipelines)](https://dev.azure.com/event-maker-telegram-bot/Core/_build/latest?definitionId=2&branchName=feature%2Fissue-2-add-azure-pipelines)
```

Comment:
This badge is related to this feature branch, will it be updated accordingly with each PR? We need it to display what happens on `master` branch. May you please check / confirm how it works?

Reply:
@bmind12 good catch, now branch name is `feature%2Fissue-2-add-azure-pipelines`, will change it